### ISSUE TITLE
Shot In the Dark?

### ISSUE THREAD

**Issue by idkjs** [Jasonette/Jasonette-Web#6]
Let say I give you these json files from [c8r/lab](https://github.com/c8r/lab)'s compositor.io design tool: [lab.json](https://github.com/c8r/lab-portfolio-kit/blob/master/lab.json) and [theme.json](https://github.com/c8r/lab-portfolio-kit/blob/master/theme.json)

Seems like that with a minor change, should translate to Jasonette-Web, no?

How do I label this as not an issue but a suggestion?

**Comment by gliechtenstein** [comment]
Yes they're all JSON so it's a matter of transforming one JSON to another.

But I think the critical part is parsing the JSX attributes. Probably need to use a JSX parser to parse it and transform it into JSON format.

I personally won't spend my time for this, but if anyone tries this and shares it here, I totally welcome.

p.s.

BTW what would the use case for this be?

**Comment by idkjs** [comment]
tldr: I hate css and want to reuse the compositor.io json...

Just thinking out loud around json and css. compositor outputs the components in json so, on principle, should be able to do this. General idea is to convert the json to use in a reasonml project so of course, i thought about celljs/jason thinking if i get it done for one, should not be to far from making it usable in this.
